Makefile

```diff
@@ -113,3 +113,15 @@ update-third-party:
 .PHONY: refresh-sample-testdata out/bincapz
 refresh-sample-testdata: out/bincapz
 	./samples/refresh-testdata.sh ./out/bincapz
+
+.PHONY: archive-samples
```

Comment:
I'd suggest to add this targets to the `make test` target. So nobody gets an error if they don't run them first.

Reply:
Definitely a good idea!